docs/advanced_topics/formbuilder_routablepage_redirect.md

```diff
@@ -0,0 +1,50 @@
+# How to… use a redirect with Form builder to prevent double submission
+
+It is common for form submission HTTP responses to be a `302 Found` temporary redirection to a new page, so that a user who hits refresh doesn't submit their information again, because the URL has changed.
```

Comment:
Maybe reverse the wording
```suggestion
When a user submits a form, if they refresh the page it will re-submit their information again, to work around this it is common for form submission HTTP responses to be a `302 Found` temporary redirection to a new page.
```

Reply:
I like your suggestion, however I think it inadvertently suggests that Wagtail FormPages' default behaviour is the correct one. I have retained my ordering to emphasise that FormPages are in the deficient minority here, but separated the sentences as you had, which is much clearer.